src/core/lib/BasicOrderFulfiller.sol

```diff
@@ -1024,6 +1006,21 @@ contract BasicOrderFulfiller is OrderValidator {
                 OrderFulfilled_consideration_body_offset
             )
 
+            // Set a length of 1 for the offer array.
+            mstore(add(eventDataPtr, 0x80), 1)
+
+            // Write itemType to the SpentItem struct.
+            mstore(add(eventDataPtr, 0xa0), offeredItemType)
+
+            // Copy calldata region with (offerToken, offerIdentifier,
+            // offerAmount) from OrderParameters to (token, identifier,
+            // amount) in SpentItem struct.
+            calldatacopy(
+                add(eventDataPtr, 0xc0),
+                BasicOrder_offerToken_cdPtr,
+                ThreeWords
+            )
```

Comment:
note for myself: these are all shifted by `0x80` compared to the previous version because old `eventConsiderationArrPtr` used `OrderFulfilled_offer_length_baseOffset = 0x200`, whereas this `eventDataPtr` only uses `OrderFulfilled_baseOffset = 0x180`. The diff is the 0x80.
Then we write to 0x80+0, 0x80+0x20, 0x80+0x40